Dockerfile

```diff
@@ -42,7 +42,7 @@ RUN curl -fL -o /usr/local/bin/docker-compose \
  # Download glibc compatible musl library for Docker Compose, see:
  # https://github.com/docker/compose/pull/3856
  && curl -fL -o /etc/apk/keys/sgerrand.rsa.pub \
-         https://raw.githubusercontent.com/sgerrand/alpine-pkg-glibc/master/sgerrand.rsa.pub \
+         https://alpine-pkgs.sgerrand.com/sgerrand.rsa.pub \
```

Comment:
@tyranron The [reason](https://github.com/sgerrand/alpine-pkg-glibc/commit/6dffcfc309e3ef7ec9ba55aaf582ab2f202906bd) of URL changes.

Reply:
@tyranron Maybe it may make sense to indicate URL changes in commit?

```
Upgrade Helm to 2.10.0 version (#16)

- change URL of sgerrand.rsa.pub
```

Reply:
@alnikyur nope